<?xml version="1.0" encoding="UTF-8"?>
<testsuites name="Mocha Tests" time="0.0000" tests="1" failures="1">
  <testsuite name="Root Suite" timestamp="2024-09-05T20:43:16" tests="0" file="cypress/integration/plugins/index-management-dashboards-plugin/force_merge.js" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="force_merge" timestamp="2024-09-05T20:43:16" tests="0" time="0.0000" failures="0">
  </testsuite>
  <testsuite name="force merge" timestamp="2024-09-05T20:43:17" tests="1" time="0.0000" failures="1">
    <testcase name="force_merge force merge force merge data stream / index / alias successfully" time="0.0000" classname="force merge data stream / index / alias successfully">
      <failure message="Timed out retrying after 60000ms: Expected to find content: &apos;Configure source index&apos; but never did." type="AssertionError"><![CDATA[AssertionError: Timed out retrying after 60000ms: Expected to find content: 'Configure source index' but never did.
    at Context.eval (http://localhost:5601/__cypress/tests?p=cypress/integration/plugins/index-management-dashboards-plugin/force_merge.js:205:10)]]></failure>
    </testcase>
  </testsuite>
</testsuites>